.travis.yml

```diff
@@ -5,6 +5,8 @@ matrix:
         - php: 5.3.3
           env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
         - php: 5.3
+        - php: 5.3
```

Comment:
But in my understanding, the deps versions will defer between `PHP 5.4 without --prefer-lowest` and `PHP 5.3 without --prefer-lowest`. I think we should test both. no?